src/components/TopologyCards/TopologyCardSmall.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "../Icon";
+import { cardStatusBorderTop } from "../../utils/common";
+
+export const TopologyCardSmall = ({ title, status }) => (
+  <div
+    className={`rounded-8px mb-4 shadow-card border-t-6 ${cardStatusBorderTop(
+      status
+    )} card cursor-pointer`}
+  >
+    <div className="flex flex-row flex-nowrap rounded-t-8px pt-2.5 pr-2.5 pb-4 pl-4 bg-white">
+      <div className="text-gray-color pt-3 mr-2.5 flex-initial max-w-1/4">
+        <h6 className="text-gray-color text-2xsi">http://</h6>
```

Comment:
```suggestion
        <span className="text-gray-color text-2xsi">http://</h6>
```

h1-h6 - заголовки. это не заголовок